tensorboard/plugins/text/summary_v2.py

```diff
@@ -26,6 +26,40 @@
 def text(name, data, step=None, description=None):
     """Write a text summary.
 
+    See also `tf.summary.scalar`, `tf.summary.SummaryWriter`, `tf.summary.image`.
+
+    Logs text tensor values for later visualization and analysis in TensorBoard.  Like
+    `tf.summary.scalar` points, text points are each associated with a `step` and a
+    `name`.  All the points with the same `name` constitute a time series of text values.
```

Comment:
Nice.  Much of this paragraph applies to tf.summary.image, so I will copy/paste if that is alright :)